### PR TITLE
Feature/prepare components for file sending

### PR DESCRIPTION
**Pull request by siepra**

### CODE REVIEW

File: packages/desktop/src/renderer/components/Channel/File/UploadingPreview.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+const UploadFilesPreviewsComponent: React.FC<UploadFilesPreviewsProps> = ({
+  filesData,
+  removeFile
+}) => {
+  const classes = useStyles({})
+  return (
+    <div className={classes.inputFiles}>
+      {Object.entries(filesData).map(fileData => (
+        <FilePreviewComponent fileData={fileData[1]} onClick={() => removeFile(fileData[0])} />
```
Comment (vinkabuki) [inline packages/desktop/src/renderer/components/Channel/File/UploadingPreview.tsx:137]: I'm not sure I understand this, could you explain why we passing data of second element while attaching onClick event to remove first one? Or is it mistake?

**Review by vinkabuki — APPROVED** [review]

File: packages/desktop/src/renderer/components/Channel/File/UploadingPreview.tsx (hunk 1 above)
Comment (siepra) [reply packages/desktop/src/renderer/components/Channel/File/UploadingPreview.tsx:137]: That's not my change, I was only moving this file to another directory

Comment (siepra) [reply packages/desktop/src/renderer/components/Channel/File/UploadingPreview.tsx:137]: I either works or there's a bug in the latest release

Comment (vinkabuki) [reply packages/desktop/src/renderer/components/Channel/File/UploadingPreview.tsx:137]: alright then

**Pull request merged**